src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -175,6 +182,15 @@ function local_branches_without_main {
 }
 
 
+function local_branches_with_gone_remote {
+  git branch -vv              |  # get all branches
+  tr '*' ' '                  |  # remove the active branch marker
+  grep  "\[origin/.*: gone\]" |  # leave only branches whose remove is marked as gone
```

Comment:
remove -> remote
Would also prefer to use the delete tracking branch terminology though